Jenkinsfile_CNP

```diff
@@ -7,13 +7,15 @@ import uk.gov.hmcts.contino.HealthChecker
 
 properties([
         parameters([
-                string(name: 'PRODUCT_NAME', defaultValue: 'ccd', description: ''),
+                choice(name: 'PRODUCT_NAME', choices: 'probate\nccd', description: 'product name like probate,ccd, div.'),
+                choice(name: 'DATA_BASE_NAME', choices: 'probatemandb\nccd-data-store-api', description: 'DB to which logstash attached (to read keys from valut)'),
```

Comment:
@dwaynebailey thinking about what you said, I think it would be better not to give the user the ability to select the db. CCD will always connect to its db and probate to probates. We should just hardcode that. 
This would prevent mistakes. What do you think?

Reply:
I agree, a simple lookup prevent any potential mistake.

Reply:
@reddy1praveen can you then remove the database dropdown please?